docs/API.md

```diff
@@ -58,6 +58,65 @@ Regex syntax: `(...)`.
 
 Captures, also known as capturing groups, extract and store parts of the matched string for later use.
 
+### `nonCaptureGroup()`
+
+```ts
+function nonCaptureGroup(
+  sequence: RegexSequence
+): NonCaptureGroup
+```
+
+Regex syntax: `(?:...)`.
+
+Groups multiple tokens together without creating a capture group.
+
+### `positiveLookahead()`
+
+```ts
+function positiveLookahead(
```

Comment:
How about `lookahead` for positive one and `negetiveLookahead` for negative one, mimicking [Swift Regex Builder](https://github.com/apple/swift-evolution/blob/main/proposals/0351-regex-builder.md#anchors-and-lookaheads) in naming?